app/presenters/base_presenter.rb

```diff
@@ -14,8 +14,16 @@ def self.presents(name)
 
   private_class_method :presents
 
-  private
+  def format_date(date)
+    date.strftime(Settings.date_format) rescue nil
+  end
 
+  def date_format(options={})
+    options.assert_valid_keys(:include_time)
+    options[:include_time] ? Settings.date_time_format : Settings.date_format
+  end
+
+  private
 
```

Comment:
Wouldn't it be better to scrap format_date, and just use date_format everywhere?